src/routes/votings/[id]/+page.svelte

```diff
@@ -1,9 +1,495 @@
+				<div class="flex items-center gap-x-1">
+					<p class="heading-02">สมาชิกสภา</p>
+					<p class="body-02">{totalVote}</p>
+				</div>
+				{#each data.results as result}
+					<div class="flex items-center gap-x-1">
+						<div class="w-4 h-4 rounded-sm {getVoteColor(result.voteOption)}" />
+						<p class="heading-02">
+							{typeof result.voteOption === 'object' ? result.voteOption.label : result.voteOption}
+						</p>
+						<p class="body-02">{result.total}</p>
+					</div>
+				{/each}
+			</div>
+		</div>
+		<div class="flex w-full h-[50px] my-2">
+			{#each data.results as result}
+				<div
+					class="rounded-sm h-full {getVoteColor(result.voteOption)}"
+					style="width: {getWidthPercent(result.total, totalVote)}%"
+				/>
+			{/each}
+		</div>
+		<div class="flex flex-col md:flex-row items-start md:items-center gap-x-2">
+			<VotingResultTag result={data.voting.result} isLarge />
+			<p class="heading-01 mt-2 md:mt-0 text-gray-60">เงื่อนไข</p>
+			<span class="body-01 flex items-center gap-x-1 text-gray-60">
+				<p class="heading-01">1.</p>
+				ได้เสียงเกินกึ่งหนึ่งของสภา
+			</span>
+			<span class="body-01 flex items-center gap-x-1 text-gray-60">
+				<p class="heading-01">2.</p>
+				ได้เสียงฝ่ายค้านอย่างน้อย 20%
+			</span>
+			<span class="body-01 flex items-center gap-x-1 text-gray-60">
+				<p class="heading-01">3.</p>
+				ได้เสียง สว. อย่างน้อย 1 ใน 3
+			</span>
+		</div>
+		<Modal passiveModal bind:open modalHeading="งดออกเสียง vs. ไม่ลงคะแนน" on:open on:close>
+			<p class="body-01">
+				งดออกเสียง เกิดจากการเสียบบัตรยืนยันตัวตน และลงคะแนนว่า “งดออกเสียง” <br />
+				ไม่ลงคะแนน เกิดจากการเสียบบัตรยืนยันตัวตนแล้ว แต่ไม่กดลงคะแนน
+			</p>
+		</Modal>
+		<button
+			class="cursor-pointer helper-text-01 mt-2 text-blue-60 underline text-left w-[260px]"
+			on:click={() => {
+				open = true;
+			}}
+		>
+			งดออกเสียง และ ไม่ลงคะแนน ต่างกันอย่างไร?
+		</button>
+		<div class="flex flex-col w-full mt-20">
+			<div class="flex flex-col md:flex-row items-start md:items-center gap-x-3">
+				<h1 id="byParty" class="fluid-heading-04">ผลการลงมติรายสังกัด</h1>
+				<div class="flex items-center">
+					<ToggleSkeleton class="outline-none" on:click={() => (isViewPercent = !isViewPercent)} />
+					<p class="body-compact-01">ดูร้อยละ</p>
+				</div>
+			</div>
+			<p class="label-01">*หมายเหตุ: ข้อมูลสังกัด ยึดตามวันที่ลงมติ</p>
+		</div>
+		<div class="flex flex-col md:flex-row w-full mt-4 mb-10 gap-x-8">
+			{#each data.resultsByAffiliation as { affiliationName, resultSummary, byParties }}
+				{@const totalAffVote = resultSummary.reduce((acc, vote) => acc + vote.total, 0)}
+				{@const totalHighestVote = data.relatedBill
+					? resultSummary.find((vote) => vote.voteOption === 'เห็นด้วย')?.total || 0
+					: resultSummary.reduce(
+							(max, current) => (current.total > max.total ? current : max),
+							resultSummary[0]
+					  ).total}
+				<div
+					class="flex flex-col w-full md:w-1/{data.resultsByAffiliation
```

Comment:
Tailwind can't process JS expression, so `md:w-1/{data.resultsByAffiliation.length}` won't work. You need to use CSS in `style` instead.